src/commands/talk.ts

```diff
@@ -11,28 +11,25 @@ import {
 	NoSubscriberBehavior,
 	entersState,
 } from '@discordjs/voice';
-import { say, Speaker } from 'dectalk';
+import dectalk from 'dectalk-tts';
 import { writeFileSync, createReadStream } from 'node:fs';
 import { fileSync } from 'tmp';
 
 import { info } from '../logger.js';
 
+const SPEAKERS = ['PAUL', 'BETTY', 'HARRY', 'FRANK', 'DENNIS', 'KIT', 'URSULA', 'RITA', 'WENDY'];
+
 const builder = new SlashCommandBuilder()
 	.setName('talk')
 	.setDescription('Uses Dectalk to speak the given message')
```

Comment:
Might be a good idea to make clear in the command description too that we send the data elsewhere:

```suggestion
	.setDescription('Uses Dectalk to speak the given message. (Input will be processed at https://tts.cyzon.us)')
```

or maybe 

```suggestion
	.setDescription('Uses Dectalk via https://tts.cyzon.us to speak the given message')
```